box.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
You can optimise it later but I would recommend to enable compression (`"compression": "GZ"`)

Reply:
That failed for me before, but now works. Thanks :+1:

Reply:
IIRC this can fail if the PHAR is too big (with the current implementation) but this should be ok unless you have over 10K files

Reply:
There was missing argument in constructor error, but I can reproduce it